be/src/exec/workgroup/work_group.cpp

```diff
@@ -181,6 +187,41 @@ void WorkGroup::estimate_trend_factor_period() {
     _period_ask_chunk_num = 1;
 }
 
+bool WorkGroup::is_big_query(const QueryContext& query_context) {
+    if (_cur_query_num <= 1) {
+        return false;
+    }
+
+    // If there is only one query, do not check the big query
+    int64_t time_now = MonotonicNanos();
+    if (time_now - query_context.query_begin_time() <= config::min_execute_time * NANOS_PER_SEC) {
+        return false;
+    }
+
+    // check cpu
+    int64_t wg_growth_cpu_use_cost = total_cpu_cost() - query_context.get_init_wg_cpu_cost();
+    if (wg_growth_cpu_use_cost == 0) {
+        return false;
```

Comment:
when wg_growth_cpu_use_cost is zero, do need check io_use_ratio againt?

Reply:
Yes, there is a logic error here, I will fix it